### ISSUE TITLE
No program icon in taskbar

### ISSUE THREAD

**Issue by MegaV0lt** [meetfranz/franz#300]
### Expected Behavior
Icon should be displayerd in taskbar

### Current Behavior
No icon is displayed

### Screenshots (if appropriate):
![auswahl_20171118_14 43 57](https://user-images.githubusercontent.com/2804301/32980893-0596e478-cc6f-11e7-8c2c-15e9975280d9.png)

**Comment by alecive** [comment]
I think it's still related to how Franz manages icons in general on Linux (see #233 )

**Comment by MegaV0lt** [comment]
Well, was working in version 4.x ...

**Comment by nomike** [comment]
I'm running Ubuntu 17.10 with XFCE4 in a VM.
There you can see the default Icon displayed in the taskbar. So it doesn't look like an issue of scaling (as reported in #233) The tray icon looks fine BTW.

See this screenshot:
![image](https://user-images.githubusercontent.com/1063307/33175268-7a33029a-d05b-11e7-92d3-9c9abafd3f46.png)

**Comment by MegaV0lt** [comment]
Nope... In Version4.x it was good:
![tooltip_20171105_13 45 162](https://user-images.githubusercontent.com/2804301/33176015-d9ea6726-d05d-11e7-8fdf-fbe7a066bd5b.png)

And that is what it should look like. Your screen just shows an defect or a placeholder icon

**Comment by nomike** [comment]
Yes, v. 4 was working for me too. But v. 5 was a major rewrite/overhaul and this seems to be one of the casualties of it.

**Comment by haraldox** [comment]
@antonpirker we'll look into your PR shortly!
